kickstarts/partials/post/node.ks.erb

```diff
@@ -0,0 +1,5 @@
+# Get the NodeJS RPM repo setup
+curl --silent --location https://rpm.nodesource.com/setup_6.x | bash -
```

Comment:
Is there a better way to do this? I don't particularly like feeding links directly to bash.

If this just installs a `.repo` file then we could probably add the repo url directly to https://github.com/ManageIQ/manageiq-appliance-build/blob/master/kickstarts/partials/main/repos.ks.erb and then install the package in the package list.